reference/7.2/Microsoft.PowerShell.Core/About/about_Redirection.md

```diff
@@ -220,6 +220,52 @@ encoding. If the file has a different encoding, the output might not be
 formatted correctly. To write to files with a different encoding, use the
 `Out-File` cmdlet with its `Encoding` parameter.
 
+### Width of output when writing to a file
+
+When you are writing to a file using either `Out-File` or the redirection
+operators, PowerShell formats table output to the file based on the width of
+the console it is running within. For instance, when logging table output
+to file using a command like e.g. `Get-ChildItem Env:\Path > path.log` on a
+system where the console width is set to 80, the output in the file is
+truncated to 80 characters:
+
+```text
+Name                         Value
+----                         -----
+Path                         C:\Program Files\PowerShell\7-preview;C:\WINDOWS…
+```
+
+Considering that the console width may be set arbitrarily on systems where
+your script is run, you may prefer that PowerShell format table output to
+files based on a width that you specify instead.
+
+The `Out-File` cmdlet provides a `Width` parameter that allows you to set the
+width you would like for table output. Rather than having to add
```

Comment:
The description is very focused on table output. It makes it sound like the width only affects tables, but the width applies to any output.

Reword this to make it clearer that the width applies to all output and that you are using Format-Table as an example.

Reply:
My experiments indicate it doesn't impact Format-List output e.g.:
```powershell
$obj = [pscustomobject]@{prop1=("*" * 1000); prop2=("-" * 1000)}
$obj | format-list > obj.log
notepad .\obj.log
```
You'll see in notepad that each line is 1009 in length without doing anything to the default width. I see the same thing if I use `Format-Custom` instead of `Format-List`.   And if I send a string to output like this:
```powershell
$obj | format-list > obj.log
notepad .\obj.log
```
I see a line in Notepad that ends at col 1001.  I'll bring this up in the wg-engine discord channel to verify.